com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Changed
 
+- Updated `UnityTransport` dependency on `com.unity.transport` to 1.3.0. (#630)
```

Comment:
@kiyoji23 The PR number is "this" PR number. So it should be #2231

Reply:
oh ok got it will modify.

Reply:
done.